src/nnet3/nnet-simple-component.cc

```diff
@@ -100,9 +100,11 @@ void DropoutComponent::InitFromConfig(ConfigLine *cfl) {
   int32 dim = 0;
   BaseFloat dropout_proportion = 0.0;
   bool dropout_per_frame = false;
+  test_mode_ = false;
   bool ok = cfl->GetValue("dim", &dim) &&
     cfl->GetValue("dropout-proportion", &dropout_proportion);
   cfl->GetValue("dropout-per-frame", &dropout_per_frame);
+  cfl->GetValue("test-mode", &test_mode_);
```

Comment:
Can you comment here that it only makes sense to set test-mode in the config for testing purposes?